website/docs/src/content/component-reference/vercel-ai.mdx

```diff
@@ -0,0 +1,334 @@
+
+##### Return Type
+
+Returns a structured object matching the provided schema.
+
+#### `<Embed/>`
+
+The `Embed` component generates embeddings for a single text input, which can be used for semantic search, clustering, and other NLP tasks.
+
+```tsx
+import { Embed } from "@gensx/vercel-ai-sdk";
+import { openai } from "@ai-sdk/openai";
+
+const embeddingModel = openai.embedding("text-embedding-3-small");
+
+// Generate an embedding for a single text
+const embedding = await gsx.execute(
+  <Embed value="This is a sample text to embed" model={embeddingModel} />,
+);
+
+console.log(embedding); // Vector representation of the text
+```
+
+##### Props
+
+The `Embed` component accepts all parameters from the Vercel AI SDK's `embed` function:
+
+- `value` (required): The text to generate an embedding for
+- `model` (required): The embedding model to use (from Vercel AI SDK)
+- Plus all other parameters supported by the Vercel AI SDK
+
+##### Return Type
+
+Returns a vector representation (embedding) of the input text.
+
+#### `<EmbedMany/>`
+
+The `EmbedMany` component generates embeddings for multiple text inputs in a single call, which is more efficient than making separate calls for each text.
+
+```tsx
+import { EmbedMany } from "@gensx/vercel-ai-sdk";
+import { openai } from "@ai-sdk/openai";
+
+const embeddingModel = openai.embedding("text-embedding-3-small");
+
+// Generate embeddings for multiple texts
+const embeddings = await gsx.execute(
+  <EmbedMany
+    values={[
+      "First text to embed",
+      "Second text to embed",
+      "Third text to embed",
+    ]}
+    model={embeddingModel}
+  />,
+);
+
+console.log(embeddings); // Array of vector representations
+```
+
+##### Props
+
+The `EmbedMany` component accepts all parameters from the Vercel AI SDK's `embedMany` function:
+
+- `values` (required): Array of texts to generate embeddings for
+- `model` (required): The embedding model to use (from Vercel AI SDK)
+- Plus all other parameters supported by the Vercel AI SDK
+
+##### Return Type
+
+Returns an array of vector representations (embeddings) for the input texts.
+
+#### `<GenerateImage/>`
+
+The `GenerateImage` component generates images from text prompts using image generation models.
+
+```tsx
+import { GenerateImage } from "@gensx/vercel-ai-sdk";
+import { openai } from "@ai-sdk/openai";
+
+const imageModel = openai.image("dall-e-3");
+
+// Generate an image
+const result = await gsx.execute(
+  <GenerateImage
+    prompt="A futuristic city with flying cars and neon lights"
+    model={imageModel}
+  />,
+);
+
+console.log(result.url); // URL to the generated image
+```
+
+##### Props
+
+The `GenerateImage` component accepts all parameters from the Vercel AI SDK's `experimental_generateImage` function:
+
+- `prompt` (required): The text description of the image to generate
+- `model` (required): The image generation model to use (from Vercel AI SDK)
+- Plus all other parameters supported by the Vercel AI SDK
```

Comment:
Nit: I believe this should be "plus any other optional parameters..."

The goal here should be to document required parameters, and link to the vercel docs for other supported parameters and output types.